packages/@react-stately/virtualizer/src/useVirtualizerState.ts

```diff
@@ -73,6 +73,11 @@ export function useVirtualizerState<T extends object, V, W>(opts: VirtualizerPro
     virtualizer.afterRender();
   });
 
+  // eslint-disable-next-line arrow-body-style
+  useEffect(() => {
+    return () => virtualizer.willUnmount();
+  });
```

Comment:
```suggestion
  }, []);
```

Reply:
yikes! good catch

Reply:
also, it worries me that the tests passed with that